src/components/AudioCell.vue

```diff
@@ -436,33 +436,66 @@ const moveDownCell = moveCell(1);
 // 消去
 const willRemove = ref(false);
 const removeCell = async () => {
-  // 1つだけの時は削除せず
-  if (audioKeys.value.length > 1) {
+  let audioKeysToDelete: AudioKey[];
+  if (
+    isMultiSelectEnabled.value &&
+    store.getters.SELECTED_AUDIO_KEYS.includes(props.audioKey)
+  ) {
+    audioKeysToDelete = store.state.audioKeys.filter((audioKey) =>
+      store.getters.SELECTED_AUDIO_KEYS.includes(audioKey)
+    );
+  } else {
+    audioKeysToDelete = [props.audioKey];
+  }
+  // 全て消える場合はなにもしない
+  if (audioKeys.value.length > audioKeysToDelete.length) {
     // フォーカスを外したりREMOVEしたりすると、
     // テキストフィールドのchangeイベントが非同期に飛んでundefinedエラーになる
     // エラー防止のためにまずwillRemoveフラグを建てる
     willRemove.value = true;
 
-    const index = audioKeys.value.indexOf(props.audioKey);
-    if (index > 0) {
-      emit("focusCell", {
-        audioKey: audioKeys.value[index - 1],
-      });
-    } else {
+    if (audioKeysToDelete.includes(props.audioKey)) {
+      const firstAudioKey = audioKeysToDelete[0];
+      const firstIndex = audioKeys.value.indexOf(
+        isMultiSelectEnabled.value ? firstAudioKey : props.audioKey
+      );
```

Comment:
選択位置が飛び飛びになってた場合、消去ボタンが押されたところのあたりがフォーカスされるのが分かりやすいかも？
なのでここはより簡単にできるかも
```suggestion
      const firstIndex = audioKeys.value.indexOf(props.audioKey);
```